doc/source/installation.rst

```diff
@@ -94,6 +94,32 @@ Fedora/Rawhide:
 
      $ sudo dnf install kiwi-cli
 
+
+
+Installation for SUSE Linux Enterprise Server
+---------------------------------------------
+
+{kiwi} is available and supported for SUSE Linux Enterprise Server (SLES).
+The recommended and supported way is to install {kiwi} by using zypper.
+
+However, if your rely on some extensions for {kiwi}, either the extension
+itself or any dependencies might not be available for your service pack.
+
+If you want to proceed anyway, keep these things in mind:
+
+* Extensions that are not provided by SLES are not supported.
+* You probably need to install dependencies via :command:`pip`.
+  The :command:`pip` command installs these dependencies from PyPI
+  (the Python Package Index).
+  However, this approach will not update the RPM database.
+* Depending on your security concerns, installing Python packages
+  outside the secured SLES installation may not be desirable.
+* Python packages installed from PyPI won't contain any SUSE
+  customizations.
+* Depending on your extension and its dependencies, you might even need
```

Comment:
Yes, I was torn whether to make it general enough or based on a specific example.

I've decided to go for the more general approach because I got the impression from the Buzilla entry that these plugins/dependencies might be added/supported in the future. Maybe I got the wrong impression. Are we sure enough that this specific example is still correct in the future? That's why I took a more general approach and didn't base it on a specific example.

Of course we can add it if you think, this would be a safe bet.

Reply:
ok that makes sense to me, so let's leave it more generic. I also agree with you regarding the conversation on slack that with a specific example it might be broken if the plugin changes. As the plugin is in another git project than kiwi and this docu it might happen that there is a disconnect. Even though I think the boxbuild plugin is pretty mature and is not expected to change a lot other than bugfixes and box descriptions.